src/SymbolDirectories/include/SymbolDirectories/Manager.h

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOL_DIRECTORIES_MANAGER_H_
+#define SYMBOL_DIRECTORIES_MANAGER_H_
+
+#include <filesystem>
+#include <vector>
+namespace orbit_symbol_directories {
+
+class Manager {
```

Comment:
QtSettingsBackedDirectories?

Reply:
I do prefer to call this `Manager` or `SymbolsDirectoriesManager` if we use it solely for symbol directories. This is in line with lots of other "Managers" in Orbit, which are basically singletons that handle resources. 

In case there is another use case for saving `std::filesystem::path` in QSettings, then I could rewrite this in a more general QSettingsStringManager. But then I would also make the kSymbolDirectoriesSettingsKey a ctor parameter. I cant come up with another use case right now, so I prefer to not do this right now.